apps/web/src/components/sections/navbar.tsx

```
@@ -0,0 +1,475 @@
+                        </NavigationMenuItem>
+                    ))}
+                </NavigationMenuList>
+                <SearchButton />
+                <ThemeToggle />
+                <a className="text-white transition-colors hover:text-white/80" href="https://github.com/visulima/visulima" rel="noreferrer" target="_blank">
+                    <TwitterLogoIcon className="size-4 fill-[var(--nav-text-color)]" title="Follow us on X" />
+                </a>
+                <a className="text-white transition-colors hover:text-white/80" href="https://github.com/visulima/visulima" rel="noreferrer" target="_blank">
+                    <GitHubLogoIcon className="size-4 fill-[var(--nav-text-color)]" title="Star us on GitHub" />
+                </a>
+                <a className="text-white transition-colors hover:text-white/80" href="" target="_blank">
```

Biome (2.4.7)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

(lint/security/noBlankTarget)